test/binaryen.js/kitchen-sink.js

```diff
@@ -63,6 +63,7 @@ function test_ids() {
   console.log("BinaryenUnaryId: " + Binaryen.UnaryId);
   console.log("BinaryenBinaryId: " + Binaryen.BinaryId);
   console.log("BinaryenSelectId: " + Binaryen.SelectId);
+  console.log("BinaryenDropId: " + Binaryen.DropId);
```

Comment:
this fix looks like the line was deleted by mistake before the merge to master somehow. not sure what happened there

Reply:
`test_ids()` is relatively new (from #1269). Looks like I skipped it when it was created :(

Reply:
Heh, no problem. Now at least I know what caused that ;)